src/PrestaShopBundle/Resources/config/admin/routing_international.yml

```diff
@@ -0,0 +1,22 @@
+admin_international_translations_list:
+    path: /translations/list
+    methods:  [POST]
+    defaults:
+        _controller: PrestaShopBundle:Admin/Translations:list
+        _legacy_controller: AdminTranslations
+        theme: classic
```

Comment:
Indeed, they are not required anymore. I will remove them as well from the edit section of the BO `Translations` page.  Thanks